### PR TITLE
Docs: Paste as plain text plugin in Clipboard guide

### PR DESCRIPTION
**Pull request by Mgsy**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Docs: Paste as plain text plugin in Clipboard guide. Closes ckeditor/ckeditor5#6240.

### CODE REVIEW

File: docs/framework/guides/deep-dive/clipboard.md [hunk 1]
```diff
@@ -93,6 +93,104 @@ editor.plugins.get( 'Clipboard' ).on( 'inputTransformation', ( evt, data ) => {
 }, { priority: 'lowest' } );
 ```
 
+### Paste as plain text plugin example
```
Comment (Reinmar) [inline docs/framework/guides/deep-dive/clipboard.md]:
```suggestion
### Paste as plain text plugin example

```

File: docs/framework/guides/deep-dive/clipboard.md [hunk 2]
```diff
@@ -93,6 +93,104 @@ editor.plugins.get( 'Clipboard' ).on( 'inputTransformation', ( evt, data ) => {
...
+You can use knowledge from previous sections to create a full plugin which will allow users to paste the content as plain text while the feature is toggled on.
+If you're not familiar with creating plugins in CKEditor 5, we'd advise starting from reading {@link framework/guides/creating-simple-plugin Creating a simple plugin} guide to get a better understanding of what's going on in the code below.
```
Comment (Reinmar) [inline docs/framework/guides/deep-dive/clipboard.md]:
```suggestion

If you're not familiar with creating plugins in CKEditor 5, we'd advise starting from reading {@link framework/guides/creating-simple-plugin Creating a simple plugin} guide to get a better understanding of what's going on in the code below.
```

File: docs/framework/guides/deep-dive/clipboard.md [hunk 3]
```diff
@@ -93,6 +93,104 @@ editor.plugins.get( 'Clipboard' ).on( 'inputTransformation', ( evt, data ) => {
...
+class PastePlainTextCommand extends Command {
+	refresh() {
+		// Disable the command if the editor is in read-only mode.
+		this.isEnabled = !this.editor.isReadOnly;
+	}
+
+	execute() {
+		// Activate pasting plain text.
+		this.value = !this.value;
+		this.editor.set( 'pastePlainText', this.value );
```
Comment (Reinmar) [inline docs/framework/guides/deep-dive/clipboard.md]: Is this needed? We can read the command's value in the paste listener.

File: docs/framework/guides/deep-dive/clipboard.md [hunk 4]
```diff
@@ -93,6 +93,104 @@ editor.plugins.get( 'Clipboard' ).on( 'inputTransformation', ( evt, data ) => {
...
+	}
+}
+
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		extraPlugins: [ PastePlainText ],
```
Comment (Reinmar) [inline docs/framework/guides/deep-dive/clipboard.md]: Such a plugin cannot be actually loaded via `extraPlugins`. It must be part of a build/editor setup. In feature guides we use a bit different notation with `plugins: [ Foo, ... ]` (same for the toolbar). It'd make sense here too.

**Comment by Mgsy** [comment]
Fixed.

File: docs/framework/guides/deep-dive/clipboard.md [hunk 5]
```diff
@@ -93,6 +93,111 @@ editor.plugins.get( 'Clipboard' ).on( 'inputTransformation', ( evt, data ) => {
 }, { priority: 'lowest' } );
 ```
 
+### Paste as plain text plugin example
+
+You can use knowledge from previous sections to create a full plugin which will allow users to paste the content as plain text while the feature is toggled on.
+
+If you're not familiar with creating plugins in CKEditor 5, we'd advise starting from reading {@link framework/guides/creating-simple-plugin Creating a simple plugin} guide to get a better understanding of what's going on in the code below.
```
Comment (Reinmar) [inline docs/framework/guides/deep-dive/clipboard.md]:
```suggestion
If you are not familiar with creating plugins in CKEditor 5, we would advise starting from reading {@link framework/guides/creating-simple-plugin Creating a simple plugin} guide to get a better understanding of what's going on in the code below.
```

**Pull request merged**